lametro/events.py

```diff
@@ -424,23 +424,34 @@ def find_approved_minutes(self, event):
 
         attachment_url = self.BASE_URL + '/matters/{}/attachments'.format(matter['MatterId'])
 
-
         attachments = self.get(attachment_url).json()
 
         if len(attachments) == 0:
             raise ValueError('No attachments for the approved minutes matter')
         elif len(attachments) == 1:
             return attachments[0]
         else:
-            # this meeting had both special and regular board meeting
-            # attached to the minutes matter
-            if date == 'May 28, 2015':
-                attachment_name = 'Regular Board Meeting Minutes on May 28, 2015'
-                attachment, = [each for each in attachments
-                               if each['MatterAttachmentName'] == attachment_name]
+            # This dictionary contains a mapping of dates of events known to
+            # have more than one minutes file attached to the approval matter,
+            # to the name of the attachment representing the correct minutes
+            # file.
+            handled_cases = {
+                'May 28, 2015': 'Regular Board Meeting Minutes on May 28, 2015',
+                'September 24, 2020': 'LA SAFE Minutes - September 24, 2020',
+                'June 24, 2021': 'LA SAFE MINUTES - June 24, 2021',
+                'December 2, 2021': 'Regular Board Meeting MINUTES - December 2, 2021',
+            }
+
+            if handled_cases.get(date):
```

Comment:
the intent is slightly clearer to do `if date in handled_cases`, but don't require a change

Reply:
Great idea, done.